### PR TITLE
Beitetsu Parcel, items_usable and .lua to function

### PR DESCRIPTION
**Pull request by apatrickm**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Adds the Beitetsu Parcel to items_usable, and adds beitetsu_parcel.lua to add Beitetsu same as retail

## Steps to test these changes
Use a beitetsu parcel

### CODE REVIEW

File: sql/item_usable.sql [hunk 1]
```diff
@@ -1857,6 +1857,7 @@ INSERT INTO `item_usable` VALUES (6176,'copy_of_astral_homeland',1,1,117,0,0,0,0
 INSERT INTO `item_usable` VALUES (6177,'copy_of_life-form_study',1,1,117,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (6178,'copy_of_hrohjs_record',1,1,117,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (6179,'copy_of_the_bell_tolls',1,1,117,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (6181,'Beitetsu_Parcel',1,1,24,0,0,0,0,0);
```
Comment (WinterSolstice8) [inline sql/item_usable.sql]: lowercase please

File: scripts/globals/items/beitetsu_parcel.lua [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(4060, math.random(3, 15))
+end
+
+return item_object
```
Comment (WinterSolstice8) [inline scripts/globals/items/beitetsu_parcel.lua]: no newline at end of file

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]

File: scripts/globals/items/beitetsu_parcel.lua [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(4060, math.random(3, 15))
```
Comment (zach2good) [inline scripts/globals/items/beitetsu_parcel.lua]: Please don't use a magic number here, add an entry in `items.lua`, add the relevant require to the top of this file, and replace the magic number with `xi.items.BEITETSU`

File: scripts/globals/items/beitetsu_parcel.lua [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+end
+
+return item_object
+
```
Comment (zach2good) [inline scripts/globals/items/beitetsu_parcel.lua]: From CI:
```
scripts/globals/items/beitetsu_parcel.lua:23: Found multiple newline characters next to each other, please replace these with single newlines.
```

Comment (zach2good) [reply scripts/globals/items/beitetsu_parcel.lua]: You had it correct in this commit: https://github.com/LandSandBoat/server/pull/2677/commits/f29ac19fcaebbde2066f3cf7f5ab262e15fe0c0c

File: scripts/globals/items/beitetsu_parcel.lua [hunk 5]
```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- ID: 6181
+-- Beitetsu Parcel
+-- Breaks up a Beitetsu Parcel
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
```
Comment (zach2good) [inline scripts/globals/items/beitetsu_parcel.lua:8]:
```suggestion
require('scripts/globals/items')
require("scripts/globals/msg")
-----------------------------------
```

File: scripts/globals/items/beitetsu_parcel.lua [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(xi.items.BEITETSU, math.random(3, 15))
```
Comment (zach2good) [inline scripts/globals/items/beitetsu_parcel.lua:20]: `xi.items.BEITETSU` doesn't exist in `scripts/globals/items.lua`, you need to add this line to it:
```lua
    BEITETSU                        = 4060,
```

Comment (zach2good) [reply scripts/globals/items/beitetsu_parcel.lua:20]: While you're there, you can also add an entry for `BEITETSU_PARCEL` (id 6181)

**Review by zach2good — CHANGES_REQUESTED** [review]

File: scroll_of_arise.lua [hunk 7]
```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- ID: 5101
+-- Scroll of Arise
+-- Teaches the white magic Arise
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return target:canLearnSpell(494)
```
Comment (WinterSolstice8) [inline scroll_of_arise.lua]: 494 -> xi.magic.spell.ARISE

File: scroll_of_inundation.lua [hunk 8]
```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- ID: 5106
+-- Scroll of Inundation
+-- Teaches the white Inundation
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return target:canLearnSpell(879)
```
Comment (WinterSolstice8) [inline scroll_of_inundation.lua]: 879 -> xi.magic.spell.INUNDATION

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]

**Review by WinterSolstice8 — APPROVED** [review]
LGTM though needs merge conflict fixed

**Comment by zach2good** [comment]
These are not relevant to this PR and should be removed:
- scroll_of_arise.lua
- scroll_of_inundation.lua

**Comment by apatrickm** [comment]
The other 2 scroll lua's were added because I can only put in 1 request at a time

**Comment by zach2good** [comment]
Replacing this PR with https://github.com/LandSandBoat/server/pull/2819, https://github.com/LandSandBoat/server/pull/2821, and https://github.com/LandSandBoat/server/pull/2820

**Comment by zach2good** [comment]
In future, please look up how to use git and GitHub from your desktop, not the web client, and how to squash your commits together (or rebase, if you're feeling brave)

**Pull request closed without merging**